src/edu/csus/ecs/pc2/clics/API202306/CLICSContestInfo.java

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.model.ContestInformation;
+import edu.csus.ecs.pc2.core.model.ContestTime;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.core.scoring.DefaultScoringAlgorithm;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+
+/**
+ * CLICS Contest Info.
```

Comment:
The 2023-06 spec does not include (as far as I can tell) any endpoint or similar structure called "Contest Info".  It would be very helpful if the class header at least contained an explanation of what component of the CLICS API this class corresponds to.  Even better would be, if this class directly corresponds to some CLICS component, that the class name match whatever that component name is.

Reply:
IIRC, I named it that because it is not really "contests" (plural).  The ContestService makes an array of ContestInformation objects, then serializes that into the array of all contests (actually only 1 in our case).  
I called it ContestInformation because there are 2 other "Contest" related clics objects:  ContestState and ContestAccess.  So, I just decided to call it ContestInformation, which is what the description for the Contests endpoint describes it as:  "Provides information on the current contest".  I'll add a comment to the class.